src/main/java/com/puppycrawl/tools/checkstyle/ConfigurationLoader.java

```diff
@@ -73,31 +73,31 @@
 
     /** The public ID for version 1_0 of the configuration dtd. */
     private static final String DTD_PUBLIC_ID_1_0 =
-        "-//Puppy Crawl//DTD Check Configuration 1.0//EN";
+        "-//Checkstyle//DTD Checkstyle Configuration 1.0//EN";
```

Comment:
See comment at https://github.com/checkstyle/checkstyle/pull/5869#issuecomment-392934305 .
If we are dropping old name, then this means we won't support old DTD names.

Reply:
good catch.

I updated issue description with "attention" note and explanation.

@subkrish , we need to keep DTD_PUBLIC_ID_1_0 unchanged and all other variable in this class.
Please create new variables DTD_PUBLIC_CS_ID_X_X with new values and add new keys for same value at method `createIdToResourceNameMap` of `ConfigurationLoader`.

The same should be done in other classes `*Loader`where "Puppy Crawl" should stay and new filed with proper value be added:
```
$ grep -Rl ".dtd\";" --include="*.java" *
src/main/java/com/puppycrawl/tools/checkstyle/XmlLoader.java
src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControlLoader.java
src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionsLoader.java
src/main/java/com/puppycrawl/tools/checkstyle/ConfigurationLoader.java
src/main/java/com/puppycrawl/tools/checkstyle/PackageNamesLoader.java
```